<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFormClass</class>
 <widget class="QMainWindow" name="UserFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidgetItems">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSearch">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Add new item</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditCategory"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditPrice"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonAdd">
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetOffers">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBid">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBid">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UserForm.qrc"/>
 </resources>
 <connections/>
</ui>
